app/src/components/main/analytics/events/common/testItemPages/actionEventsCreators.js

```diff
@@ -146,16 +146,14 @@ export const getRefineFiltersPanelEvents = (category) => ({
   }),
 });
 
-export const getClickOnPlusMinusEvents = (category) => (state) => ({
-  category,
-  action: 'Click on Bread Crumb icon Plus/Minus',
-  label: state ? 'Minus' : 'Plus',
+export const getClickOnPlusMinusEventCreator = (category) => (expanded) => ({
```

Comment:
Be as kind as to place these events under GA-4 category.

Reply:
Done